upgrade/sql/8.0.0.sql

```diff
@@ -48,3 +48,9 @@ ALTER TABLE `PREFIX_product_shop` MODIFY COLUMN `redirect_type` ENUM(
 /* PHP:ps_800_add_security_tab(); */;
 
 ALTER TABLE `PREFIX_order_detail` MODIFY COLUMN `product_name` TEXT NOT NULL;
+
+ALTER TABLE `PREFIX_product` ADD `unit_price` decimal(20, 6) NOT NULL DEFAULT '0.000000' AFTER `unity`;
+ALTER TABLE `PREFIX_product_shop` ADD `unit_price` decimal(20, 6) NOT NULL DEFAULT '0.000000' AFTER `unity`;
+
+UPDATE `PREFIX_product` SET `unit_price` = IF (`unit_price_ratio` != 0, `price` / `unit_price_ratio`, 0);
+UPDATE `PREFIX_product_shop` SET `unit_price` = IF (`unit_price_ratio` != 0, `price` / `unit_price_ratio`, 0);
```

Comment:
Shouldn't we use the PrestaShop round type here?

Reply:
I'm not sure, the mysql column size will automatically truncate the values anyway. As far as I know the round parameter are used during computation to decide how to round the value for display but not in the column updates. At least for this new column I didn't integrate some rounding before DB update in order to keep as many decimals as possible.